src/main/java/ChatBot.java

```diff
@@ -0,0 +1,158 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ChatBot {
+    private String chatbotName;
+    private List<Task> tasks;
+    private Scanner in;
+
+    public ChatBot(String chatbotName) {
+        this.chatbotName = chatbotName;
+        this.tasks = new ArrayList<>();
+        this.in = new Scanner(System.in);
+    }
+
+    public void startChat() {
+        printWelcomeMessage();
+
+        while (true) {
+
+            String command = in.nextLine().trim();
+
+            try {
+                if (command.equalsIgnoreCase("bye")) {
+                    printFormattedMessage("Bye. Hope to see you again soon!");
+                    break;
+                } else if (command.equalsIgnoreCase("list")) {
+                    printTaskList();
+                } else if (command.startsWith("mark ")) {
+                    markTaskAsDone(command.substring(5));
+                } else if (command.startsWith("unmark ")) {
+                    unmarkTaskAsDone(command.substring(7));
+                } else if (command.startsWith("todo ")) {
+                    addTask(command.substring(5), "todo");
+                } else if (command.startsWith("event ")) {
+                    addTask(command.substring(6), "event");
+                } else if (command.startsWith("deadline ")) {
+                    addTask(command.substring(9), "deadline");
+                } else {
```

Comment:
good, you can improve this by reducing the magic literals in the code. 
https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers